lib-injection/host_inject.rb

```diff
@@ -0,0 +1,7 @@
+# This file's intent is to be parseable and executable by all ruby versions
+# to call into the main one only for versions for which it is known-compatible
+# with at the language level.
+
+if RUBY_VERSION >= '2.5.'
```

Comment:
@TonyCTHsu since Vaccine is testing down to 2.3, should we lower this to 2.3?

Reply:
We can do that, but vaccine is testing the last patch version of 2.3

```
% docker run --rm ghcr.io/datadog/images-rb/engines/ruby:2.3 bash -c 'ruby -e "puts RUBY_VERSION"'
2.3.8
```

Reply:
Technically true, although as far as parsing/processing goes 2.3.0 to 2.3.8 are the same, so I expect it to be OK.